tensorflow/training/docker/2.3.0/py3/cu102/Dockerfile.gpu

```diff
@@ -27,7 +27,8 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.3_aws/20200902-124016/gpu/py37/tensorflow_gpu-2.3.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.3_aws_hash/20200924-012425/gpu/py37/tensorflow_gpu-2.3.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_HASH=sha256:618a53167cd5ad2a221ecea4aa16f8fb040c4d12d80e6aa087cf8eceab9f759e
```

Comment:
Since this TF_HASH will have to be modified every time the TF_URL is updated, can the hash be maintained alongside the TF_URL file at the same location?